### PR TITLE
constitution: amend amendment and dissolution processes

### PR DESCRIPTION
**Pull request by chriskuehl**
Proposed amendment by me. Voting will take place via email under section 8.1.1.2.

### CODE REVIEW

**Comment by Ranigad** [comment]
Do we express our affirmation in reply to this email or will there be a
separate one

**Comment by chriskuehl** [comment]
i will send another email shortly

File: ocfweb/docs/docs/docs/constitution.md [hunk 1]
```diff
@@ -240,14 +242,29 @@ constitution into compliance with requirements set by the University.
...
+*[This section was modified by amendment on May xx, 2016.]*
 
 #### 8.2 Bylaws
 Bylaws may be created or modified as needed, through the [OCF Decision Making Process](#ocf_decision_making_process).
 
 
-### 9  Dissolution
-The assets of the OCF constitute a continuing trust for the benefit of all members of the Berkeley campus community interested in open computing facilities.  In the event of the dissolution of the OCF for whatever reason, the assets, after payment or adequate provision for payment of all outstanding debts and obligations of the OCF shall be transferred to a non-profit fund, foundation or corporation which is organized and operated exclusively for the purposes for which the OCF was founded.  All unspent ASUC funds shall remain the property of the ASUC.
+### 9  Dissolution    {amended-2016-05-xx-part1}
+The assets of the OCF constitute a continuing trust for the benefit of all members of the Berkeley campus community interested in open computing facilities.
```
Comment (matthew-mcallister) [inline ocfweb/docs/docs/docs/constitution.md]: Line length

Comment (chriskuehl) [inline ocfweb/docs/docs/docs/constitution.md]: most of our constitution is not wrapped. i'm probably going to go fix that after this to make it consistently line wrapped.

in this case i didn't do it on that line to keep the diff more readable

File: ocfweb/docs/docs/docs/constitution.md [hunk 2]
```diff
@@ -240,14 +242,29 @@ constitution into compliance with requirements set by the University.
...
+
+If the organization is ASUC or GA Sponsored, all unspent ASUC funds shall
```
Comment (matthew-mcallister) [inline ocfweb/docs/docs/docs/constitution.md]: Do we need to say "_if_ the organization is sponsored" in our own constitution when we know we _are_ sponsored?

Yes, I know it's verbatim from the template.

Comment (chriskuehl) [inline ocfweb/docs/docs/docs/constitution.md]: i very much wish we didn't, i agree that it looks ridiculous

**Pull request merged**